.env.example

```diff
@@ -71,3 +71,5 @@ CDN_DOMAIN=cdn.climatepolicyradar.org
 # Pipeline settings
 PIPELINE_BUCKET="a-bucket-for-testing-cpr-endpoints"
 
+# Default to CPR theme
+THEME=cpr
```

Comment:
May as well add ROBOTS, and HOSTNAME as well

Reply:
Will do this as a follow-up